.gitignore

```diff
@@ -18,6 +18,9 @@
 
 # Various temporary generated files
 *.tmp
+*.dep
```

Comment:
same comment as above - if you plan on submitting this, you should remove this set of changes.